lib/car.rb

```diff
@@ -0,0 +1,27 @@
+class Car
+  attr_reader :make_model, :monthly_payment, :loan_length
+
+  def initialize(make_model, monthly_payment, loan_length)
+    @make_model = make_model
+    @monthly_payment = monthly_payment
+    @loan_length = loan_length
+  end
+
+  def make
+    make_model.split(" ")
+    car_make = make_model.split[0]
+    p car_make
+
+  end
+
+  def model
+    make_model.split(" ")
+    car_model = make_model.split[1]
+    p car_model
+  end
```

Comment:
We don't want to print these values to the console
```suggestion
  def make
    make_model.split(" ")
    car_make = make_model.split[0]
  end

  def model
    make_model.split(" ")
    car_model = make_model.split[1]
  end
```